migrate/20240722_add_project_reputation.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_enum(:project_reputation, %w[new verified])
```

Comment:
We don't have to do it in this PR. In future, We might add `fraud` option as well to block new resource creation in the time between we block the account and we suspect it's a fraud.